frontend/src/routers/OrgRouter/DefaultWorkspaceRouter.tsx

```diff
@@ -11,7 +11,7 @@ import { Navigate, useMatch } from 'react-router-dom'
 export const DefaultWorkspaceRouter = () => {
 	const { isLoggedIn } = useAuthContext()
 
-	const workspaceMatch = useMatch('/w/:workspace_id/:page_id')
+	const workspaceMatch = useMatch('/w/:page_id')
 	const pageId = workspaceMatch?.params.page_id ?? ''
```

Comment:
nit: this is getting the `workspaceId`, not `pageId`? should this just be `const workspaceMatch = useMatch('/:page_id')` since  it is relative to the `/w/X` route?